libs/live-network/src/network.ts

```diff
@@ -105,6 +105,17 @@ export const errorInterceptor = (error: InterceptedError): InterceptedError => {
 axios.interceptors.request.use(requestInterceptor);
 axios.interceptors.response.use(responseInterceptor, errorInterceptor);
 
+// not react native
+if (
+  !(typeof navigator !== "undefined" && navigator.product === "ReactNative")
```

Comment:
ideally this should happen in LLM directly. in future, let's see if we can figure out a way to make a library having a "react-native entry point" because i'm not sure polyfill should live in end libraries. ( cc @haammar-ledger  @hzheng-ledger  )